lib/fathead/perl_doc/cover/functions.txt

```diff
@@ -0,0 +1,275 @@
+-A
```

Comment:
@jbarrett Fair enough! So I'm guessing these are generated elsewhere then hard-coded into the repo? Ping me when you've got a ready draft for the docs, I'd be interested to read some of the nitty-gritty details.